packages/formation/sass/base/_b-variables.scss

```diff
@@ -124,8 +132,6 @@ $color-sidebar-nav-bg:   rgba($color-black, 0.25);
 
 $color-orange: #EB7F29;
 $color-gold-darker: #988530;
-$color-light-blue: #0071bb;
-$color-old-browser-background-start: #0071bb;
 $color-old-browser-background-end: #112e51;
```

Comment:
I think `#112e51` is the old hex value for `color-blue-darkest`. This should be updated to use either `$color-blue-darket`, or its hex value, `#162e51`